src/View/XmlView.php

```diff
@@ -151,10 +148,6 @@ protected function _serialize($serialize)
             $options['pretty'] = true;
         }
 
-        if (isset($options['return']) && strtolower($options['return']) === 'domdocument') {
```

Comment:
Both `DOMDocument` and `SimpleXMLElement` have `saveXML()` method, so this check was redundant.